package.json

```diff
@@ -46,7 +46,7 @@
     "watchify": "^3.7.0"
   },
   "scripts": {
-    "prepublish": "babel ./lib --out-dir ./dist",
+    "build": "babel ./lib --out-dir ./dist",
```

Comment:
I updated this just so we do a yarn build for the dist folder which is what we use. I didn't like using "prepublish"

Reply:
yeah this is actually an npm hook but since we don't have an npm repository it's better to switch the name